charts/jenkins/templates/jenkins-backup-cronjob.yaml

```diff
@@ -60,7 +60,11 @@ spec:
             - --container
             - jenkins
             - --path
+            {{- if .Values.backup.onlyJobs }}
+            - {{ .Values.controller.jenkinsHome }}/jobs
```

Comment:
is it worth making this configurable? it is possible to change it's location in Jenkins:
https://www.jenkins.io/doc/book/managing/system-properties/#jenkins-model-jenkins-buildsdir

(we change our workspace dir to a faster disk)

Reply:
> > The only thing needing backup are the job logs and artifacts (so the /job folder).
> 
> you can also external artifact storage (e.g. azure / aws) so that you don't need to back them up:
> 
> https://www.jenkins.io/doc/developer/extensions/jenkins-core/#artifactmanagerfactory
Indeed we are actually using Azure Artifacts. All of our pipelines get created via a job dsl script located in git too, so not even the pipeline configs would need backup. Still the pipeline state and logs need some kind of backup.



> is it worth making this configurable? it is possible to change it's location in Jenkins:
> https://www.jenkins.io/doc/book/managing/system-properties/#jenkins-model-jenkins-buildsdir
> 
> (we change our workspace dir to a faster disk)

But how do you backup the folder on the other disk? 
For me this would need some 3rd party tools / a custom setup, that I then can't automate using only the Jenkins helmchart. So in the end I could just as well use this tool to only backup the "default" /job folder. But I think the goal should be to enable a full Jenkins as code via the helmchart and for me that includes the backups too.

A possibility I might image makes more sense: Let the user specify a custom path to backup. This would then also enable backups of job folders on another disk. What do you think?

Reply:
yes worthwhile probably but for now this should be fine to ship